src/tools/wasm-shell.cpp

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2015 WebAssembly Community Group participants
+ * Copyright 2021 WebAssembly Community Group participants
```

Comment:
this isn't needed - it's best to keep the original year in files.

Reply:
I started this rewrite in a new file, then I forgot to revert it when I replaced it with the old `wasm-shell.cpp`. :) It's back to 2015 now.